.gitignore

```diff
@@ -17,6 +17,10 @@ coverage/
 *.ipa
 *.apk
 
+# Rubbish
```

Comment:
a lot of the shit in the .gitignore is useless in our context, it was just copied and pasted off somewhere i think
probably want to clean it up

Reply:
Didnt notice. Fixed.